samples/azure/azure-service-fabric-routing/Core_8/Shared/EndpointPartitioning/HardcodeReplyToAddressToLogicalAddressFeature.cs

```diff
@@ -6,7 +6,7 @@ class HardcodeReplyToAddressToLogicalAddressFeature : Feature
     protected override void Setup(FeatureConfigurationContext context)
     {
         var settings = context.Settings;
-        var addressToLogicalAddress = new HardcodeReplyToAddressToLogicalAddress(settings.InstanceSpecificQueue());
+        var addressToLogicalAddress = new HardcodeReplyToAddressToLogicalAddress(context.InstanceSpecificQueueAddress().ToString());
```

Comment:
I don't think this is the correct approach here. `ToString` does not generate a valid transport-specific address (not sure what this sample does with the value though). The `QueueAddress` needs to be translated using `ITransportAddressResolver` which is available via DI for users.

Happy to help with this.